packages/esm-appointments-app/src/patient-appointments/patient-appointments-header.tsx

```diff
@@ -0,0 +1,40 @@
+import { useTranslation } from 'react-i18next';
+import React from 'react';
+import { ArrowLeft } from '@carbon/react/icons';
+import { Button } from '@carbon/react';
+import { useNavigate } from 'react-router-dom';
+import styles from './patient-appointments-header.scss';
+import { PatientBannerPatientInfo, PatientPhoto } from '@openmrs/esm-framework';
+
+interface PatientAppointmentsHeaderProps {
+  patient: fhir.Patient;
+}
+
+const PatientAppointmentsHeader: React.FC<PatientAppointmentsHeaderProps> = ({ patient }) => {
+  const { t } = useTranslation();
+  const navigate = useNavigate();
+  const patientName = `${patient?.name?.[0]?.given?.join(' ')} ${patient?.name?.[0].family}`;
+
+  return (
+    <div>
+      <div className={styles.patientBanner}>
+        <div className={styles.patientAvatar} role="img">
+          <PatientPhoto patientUuid={patient.id} patientName={patientName} />
+        </div>
+        <PatientBannerPatientInfo patient={patient}></PatientBannerPatientInfo>
+      </div>
+      <div className={styles.titleContainer}>
+        <Button
+          kind="ghost"
+          onClick={() => navigate(-1)}
+          renderIcon={ArrowLeft}
+          iconDescription={t('back', 'Back')}
+          size="lg">
+          <span>{t('back', 'Back')}</span>
+        </Button>
+      </div>
+    </div>
+  );
+};
```

Comment:
Just pulled this together from looking at the PatientBanner component within the Patient Chart app, @brandones  would be interested in your thoughts if this is the recommended way to use the newly-abstracted patient banner style guide functionality.

Reply:
Aw heck yeah! Love to see it. Yeah the API is simple enough that as long as you're not doing anything crazy like hacking the CSS or the patient object prop, there's not really a wrong way to use it. (famous last words)

Reply:
😅